docs/src/reference/lib/api/transforms/helpers.rst

```diff
@@ -15,3 +15,4 @@ Helpers
 .. autoclass:: KlioUpdateAuditLog()
 .. autoclass:: KlioDebugMessage()
 .. autoclass:: KlioSetTrace()
+.. autoclass:: KlioTriggerUpstream()
```

Comment:
Both this and the change right below must have been originally missed when the class was implemented. This was caught now because I tried to link to it when listing the default metric it emits.